.github/workflows/releases.yml

```diff
@@ -58,92 +58,120 @@ jobs:
           asset_name: database-migrations-${{ github.event.release.tag_name }}-linux-x64.zip
           asset_content_type: application/zip
 
-
-  publish-docker:
+  setup-tags:
     runs-on: ubuntu-22.04
-    environment: release
-    permissions:
-      id-token: write
-      contents: read
-      pull-requests: write
+    outputs:
+      database-migrations-tag: ${{ steps.setup_tags.outputs.database-migrations-tag }}
+      data-aggregator-tag: ${{ steps.setup_tags.outputs.data-aggregator-tag }}
+      gateway-api-tag: ${{ steps.setup_tags.outputs.gateway-api-tag }}
     steps:
-      - name: Dump context
-        uses: crazy-max/ghaction-dump-context@516dbb0c760f39b4cdd750ae095f1688780f68f4
-
       - uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b
-        with:
-          fetch-depth: 0
-
-      - name: Get release
+      - name: Get rele ase
         id: get_release
         uses: bruceadams/get-release@f589ce0779c7bef1faf175f7488c972eb47dc046
         env:
           GITHUB_TOKEN: ${{ github.token }}
-
-      - name: "Configure AWS credentials"
-        uses: aws-actions/configure-aws-credentials@97271860067ec931c45b8d104fbf0d15954ab85c
-        with:
-          role-to-assume: arn:aws:iam::308190735829:role/gh-dockerhub-releaser
-          aws-region: eu-west-2
-
-      - name: Read secrets from AWS Secrets Manager into environment variables
-        uses: aws-actions/aws-secretsmanager-get-secrets@287592d14d9c9c48199db83dc182ae12af3df18e
-        with:
-          secret-ids: |
-            DOCKERHUB, github-actions/rdxworks/dockerhub-images/release-credentials
-          parse-json-secrets: true
-
-      - name: Login to Docker Hub
-        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
-        with:
-          username: ${{env.DOCKERHUB_USERNAME}}
-          password: ${{env.DOCKERHUB_TOKEN}}
-
       - name: Setup tags for docker image
         id: setup_tags
         uses: ./.github/actions/set-variables
         with:
           github_event_name: ${{ github.event_name }}
           github_action_name: ${{ github.event.action}}
-
       - name: Setup application version
```

Comment:
@shambupujar  This is where the file you mentioned is created. I assume its important for the application to know its own release version. I create this file inside a different job, so I need to fetch it via artifact_restore. After that its available in the docker build context and gets copied.